src/modules/interest-rates/interest-rates.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable, InternalServerErrorException, Logger } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Equal, Repository } from 'typeorm';
+
+import { InterestRatesEntity } from './entities/interest-rate.entity';
+
+@Injectable()
+export class InterestRatesService {
+  private readonly logger = new Logger();
+  constructor(
+    @InjectRepository(InterestRatesEntity, 'mssql-cedar')
+    private readonly interestRates: Repository<InterestRatesEntity>,
+  ) {}
+
+  findAll(): Promise<InterestRatesEntity[]> {
+    try {
+      return this.interestRates.find({ where: { effectiveTo: Equal('9999-12-31 00:00:00.000') } });
```

Comment:
@avaitonis is this filter what you'd expect here? i.e. return everything that has effectiveTo set to `'9999-12-31 00:00:00.000'`

Reply:
Looks good. I assume it works. It might also work without time part, just "9999-12-31".